deploy/awstools/awstools.py

```diff
@@ -48,7 +48,7 @@ def __init__(self) -> None:
         MockBoto3Instance.base_ip += 1
         self.private_ip_address = ".".join([str((self.ip_addr_int >> (8*x)) & 0xFF) for x in [3, 2, 1, 0]])
 
-def depaginated_boto_query(client, operation, operation_params, return_key):
+def depaginated_boto_query(client: Any, operation: str, operation_params: Dict[Any, Any], return_key: str) -> List[Any]:
```

Comment:
Again, you probably want to use the generic `Mapping` for `operation_params` unless you spefically need a `Dict`.  Looking at the code, we only need `Mapping` because we are only unpacking it as kwargs to `paginate`